components/QuizCardPage/User/index.test.js

```diff
@@ -1,10 +1,8 @@
 import { render, screen } from "@testing-library/react";
-import { StyledUser } from ".";
+import User from ".";
 
-describe("StyledUser components", () => {
-  it("should render the correct text", () => {
-    render(<StyledUser>😄</StyledUser>);
-    const user1Element = screen.getByText("😄");
-    expect(user1Element).toBeInTheDocument();
-  });
+test("renders User component without crashing", () => {
+  render(<User name="John" emoji="😀" points={10} />);
+  const userName = screen.getByText("John");
+  expect(userName).toBeInTheDocument();
```

Comment:
I would expect to test the correct emoji and the number of points as well to make sure the complete component works as intended.